make_mt_IHME.py

```diff
@@ -0,0 +1,254 @@
+parser.add_argument('--entry-substr', type=str, help='Substrings, column delimited, to define an entry column. Uses OR between provided substraings.', default='val,upper,lower')
+parser.add_argument('--entry-segment-order', type=str, help='Entry columns are those that contain the substrings in --entry-subsr. They are period-delimited to represent hail structs. This comma-delimited string defines nesting order.', default='3,2,1')
+parser.add_argument('--output-table', type=str, help='Output a derivative Hail Table to a directory. If not provided, will not output. If provided, expects --filter-specific-column to select a column for the final table.', default='')
+parser.add_argument('--filter-specific-column', type=str, help='Filter the MatrixTable to a specific column for conversion to a Table. Only considered if --output-table is provided. Note that the column "Age-standardized" only contains rates.', default='Age-standardized')
+parser.add_argument('--explode-icd', type=str, help='Explode an ICD column. If not provided, will not explode. If provided, will remove all other ICD columns. Only considered if --output-table is provided.', default='')
+
+
+if __name__ == '__main__':
+    args = parser.parse_args()
+    
+    # Import
+    ht = hl.import_table(args.input, force=True, impute=True)
+    field_names = list(ht.row.keys())
+
+    entry_cols = get_entry_cols(args.entry_substr, field_names)
+    column_key, column_other_fields = get_column_cols(args.col_key, args.col_other, field_names)
+    row_key, row_other_fields = get_row_cols(args.row_key, args.row_other, 
+                                             col_fields=[column_key] + column_other_fields,
+                                             entry_fields=entry_cols, field_names=field_names)
+    
+    # Ensure that row, column, and entry schema contains appropriate names in the Hail Table.
+    all_cols = [column_key, row_key] + row_other_fields + column_other_fields + entry_cols
+    all_cols.sort()
+    if not all([col in field_names for col in all_cols]):
+        raise ValueError('All selected fields must be in the columns of the read table.')
+    if any(all_cols.count(x) > 1 for x in all_cols):
+        raise ValueError('There are duplicate items in the final set of columns. ' + \
+                        'Ensure that row items are only assigned to rows, column ' + \
+                        'items only to columns, etc.')
+
+    # Make MatrixTable with proper entry structure.
+    ht_m = make_entries_into_struct(ht, entry_cols, args.entry_segment_order, drop_old_entries=True)
+    mt = ht_m.to_matrix_table(row_key=[row_key], row_fields=row_other_fields,
+                              col_key=[column_key], col_fields=column_other_fields,
+                              n_partitions=100)
+    mt = mt.annotate_globals(year=args.year)
```

Comment:
Is this for tracking? Is the date in the filename? If so, filename might be better than a user-specified year - if a user picks a new file they might not notice this argument and not update it